scripts/docker_files/docker_file_wheelbuilder_linux/Dockerfile

```diff
@@ -1,43 +1,39 @@
-FROM quay.io/pypa/manylinux2010_x86_64
+FROM quay.io/pypa/manylinux2014_x86_64
 
-ENV MMG_ROOT=/external_libraries/mmg/mmg_5_4_1
+ENV MMG_ROOT=/external_libraries/mmg/mmg_5_5_1
 ENV LD_LIBRARY_PATH ${LD_LIBRARY_PATH}:{MMG_ROOT}/lib
 
+# Prepare package dependencies
 RUN yum install -y git wget zip
 
-RUN yum install -y libgfortran.x86_64 blas-devel lapack-devel
-
-# Prepare File system
+# Prepare Shared Volumes
 RUN mkdir -p /data_swap_guest
 
+# Prepare File System
 RUN mkdir -p /workspace/scripts; \
-	mkdir -p /workspace/kratos
+	mkdir -p /workspace/kratos; \
+    mkdir -p /workspace/boost
+
+# Install Boost (from source)
```

Comment:
Before anyone comments. I know its commented. The intention is to have boost cached in the compilation server so we don't need to redownlod it every time we build the image (because is slow as hell), but I did not have time to finish implementing.